rialto.supp

```diff
@@ -123,3 +123,14 @@
    fun:dlopen_implementation
    fun:dlopen@@GLIBC_2.34
 }
+{
```

Comment:
Have you tried calling gst_deinit in suites in which gst_init was called?

Reply:
As discussed, this didn't fix the issue but was a worthwhile improvement anyway.